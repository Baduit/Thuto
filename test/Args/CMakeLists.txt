add_executable(test_args_main.test test_args_main.cpp)
add_test(test_args_main test_args_main.test)