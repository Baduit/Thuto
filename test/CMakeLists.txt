add_subdirectory(containers)
add_subdirectory(io)
add_subdirectory(concurrency)
add_subdirectory(utils)
add_subdirectory(process)