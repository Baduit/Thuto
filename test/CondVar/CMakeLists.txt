add_executable(test_cond_var_main.test test_cond_var_main.cpp)
add_test(test_cond_var_main test_cond_var_main.test)