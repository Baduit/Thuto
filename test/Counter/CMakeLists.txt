add_executable(test_counter.test test_counter_main.cpp)
add_test(test_counter test_counter.test)
target_link_libraries(test_counter.test ${CMAKE_THREAD_LIBS_INIT})