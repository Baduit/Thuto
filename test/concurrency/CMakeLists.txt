addTest(event_queue test_event_queue.cpp)