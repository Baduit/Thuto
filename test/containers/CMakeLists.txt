addTest(vector2d test_vector2d.cpp)