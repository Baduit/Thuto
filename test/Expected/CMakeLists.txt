add_executable(test_expected_main.test test_expected_main.cpp)
add_test(test_expected_main test_expected_main.test)